packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -958,7 +963,13 @@ export class BasePickerListBelow<T, P extends IBasePickerProps<T>> extends BaseP
       <div ref={this.root} onBlur={this.onBlur}>
         <div className={classNames.root} onKeyDown={this.onKeyDown}>
           {this.getSuggestionsAlert(classNames.screenReaderText)}
-          <div className={classNames.text}>
+          <div
+            className={classNames.text}
+            aria-owns={suggestionsAvailable || undefined}
+            aria-expanded={!!this.state.suggestionsVisible}
+            aria-haspopup="true"
```

Comment:
Why is this `true` here instead of `listbox/dialog` like before?

Reply:
Good catch